docs/en_US/AdvancedFeature/MultiPhase.md

```diff
@@ -48,6 +70,10 @@ trial_end
 ```
 With this information, the tuner could know which trial is requesting a configuration, and which trial is reporting results. This information provides enough flexibility for your tuner to deal with different trials and different phases. For example, you may want to use the trial_job_id parameter of generate_parameters method to generate hyperparameters for a specific trial job.
 
-Of course, to use your multi-phase tuner, __you should add `multiPhase: true` in your experiment YAML configure file__.
 
-[ENAS tuner](https://github.com/countif/enas_nni/blob/master/nni/examples/tuners/enas/nni_controller_ptb.py) is an example of a multi-phase tuner.
+### Tuners support multi-phase experiments:
+
+[TPE](../Tuner/BuiltinTuner.md#TPE), [Random](../Tuner/BuiltinTuner.md#Random), [Anneal](../Tuner/BuiltinTuner.md#Anneal), [Evolution](../Tuner/BuiltinTuner.md#Evolution), [SMAC](../Tuner/BuiltinTuner.md#SMAC), [NetworkMorphism](../Tuner/BuiltinTuner.md#NetworkMorphism), [MetisTuner](../Tuner/BuiltinTuner.md#MetisTuner), [BOHB](../Tuner/BuiltinTuner.md#BOHB), [Hyperband](../Tuner/BuiltinTuner.md#Hyperband), [ENAS tuner](https://github.com/countif/enas_nni/blob/master/nni/examples/tuners/enas/nni_controller_ptb.py).
```

Comment:
readthedocs may not support patterns such as #TPE, #Random.

Reply:
Thanks, updated.